pubspec.yaml

```diff
@@ -40,14 +45,115 @@ dev_dependencies:
 
 flutter:
   assets:
-    - fonts/google_fonts/
-    - assets/demos/
-    - assets/icons/cupertino/
-    - assets/icons/material/
-    - assets/icons/reference/
-    - assets/icons/settings/
-    - assets/logo/
-    - assets/studies/
+    - packages/flutter_gallery_additional_assets/assets/icons/settings/settings_light.flr
```

Comment:
You don't have to specify every size, just the "size-less" version

Reply:
Done.